guides/doc-Planning_for_Project/topics/Glossary.adoc

```diff
@@ -117,6 +117,21 @@ The plug-in consists of three components: services running on {ProjectServer} an
 *Docker Tag*:: A mark used to differentiate container images, typically by the version of the application stored in the image.
 In the {ProjectWebUI}, you can filter images by tag under *Content* > *Docker Tags*.
 
+ifndef::satellite[]
+[[varl-Glossary_of_Terms-Enterprise_Linux]]
+*Enterprise Linux*:: An umbrella term for the following operating systems:
+
+* AlmaLinux
+* CentOS Linux
```

Comment:
```suggestion
```

I'd drop this because it's going to be EoL soon: https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/

But I am OK either way.

Reply:
Just saw https://github.com/theforeman/foreman-documentation/issues/2679#issuecomment-1910717289 > closing this comment :)